main.rb

```diff
@@ -0,0 +1,182 @@
+require_relative('./person')
+require_relative('./book')
+require_relative('./rental')
```

Comment:
The user is unable to create a rental. Please see the screenshot below to reference the error.
![create-rental](https://user-images.githubusercontent.com/35504149/147246819-5ef74444-fbca-4854-8b76-fe8fee946b9f.png)